src/app/Calendar/Yearly/index.styles.scss

```diff
@@ -68,3 +68,9 @@
   width: 100%;
   color: #202029;
 }
+
+@media only screen and (max-width: 426px) {
+    .yearlyMonth {
+        width: calc(100% / 2);
```

Comment:
In the case with only one month per row, the width will become 100% ( actually calc(100% / 1) )